tests/PathTest.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+class PathTest extends \PHPUnit\Framework\TestCase
+{
+    public function testResolve() : void
+    {
+        $path = __DIR__ . '/../src/SwedbankPay/Api/../';
+        $resolvedPath = \SwedbankPay\Path::resolve($path);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid using static access to class '\SwedbankPay\Path' in method 'testResolve'.](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-sdk-php/pullRequest?prid=5303907)